src/SSHTunnel.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorSSHTunnel;
+
+
+use Keboola\DbExtractorLogger\Logger;
+use Keboola\DbExtractorSSHTunnel\Exception\UserException;
+use Keboola\SSHTunnel\SSH;
+use Keboola\SSHTunnel\SSHException;
+use Retry\BackOff\ExponentialBackOffPolicy;
+use Retry\Policy\SimpleRetryPolicy;
+use Retry\RetryProxy;
+
+class SSHTunnel
+{
+
+    public const DEFAULT_LOCAL_PORT = 33006;
+
+    public const DEFAULT_SSH_PORT = 22;
+
+    public const DEFAULT_MAX_TRIES = 5;
+
+    /** @var Logger */
```

Comment:
Tady měl @odinuv dobrou poznámku, že ono to nepotřebuje přímo nějaký konkrétní logger. Potřebuje to jen 

composer require psr/log

Reply:
Což umožní to, že to můžem releasnout hned jak to bude ready a nemusíme čekat na logger...

Reply:
je to tam